int64 index
bool vote
